src/models/person.js

```diff
@@ -0,0 +1,31 @@
+export default (sequelize, DataTypes) => {
+  const Person = sequelize.define('person', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    dni: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    }
+  })
+
+  Person.associate = (models) => {
+    Person.belongsToMany(models.physicalAddress, {
+      as: 'physicalAddresses',
+      through: 'personPhysicalAddress'
+    })
+    Person.belongsTo(models.user)
```

Comment:
```suggestion
    Person.hasOne(models.user)
```
https://sequelize.org/master/manual/assocs.html
> The A.hasOne(B) association means that a One-To-One relationship exists between A and B, with the foreign key being defined in the target model (B).
> 
> The A.belongsTo(B) association means that a One-To-One relationship exists between A and B, with the foreign key being defined in the source model (A).